### ISSUE TITLE
[Enhancement] Additional icons in Action Sheets.

### ISSUE THREAD

**Issue by henrik-bd** [kirbydesign/designsystem#2351]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement

Today an Action Sheet item contains an id and a text.
We would like to be able to add an icon also.

--
Nice to have:
Individual style on the text like colour.
Disabling of items.

## Describe the solution you'd like

Today's [action-sheet page](https://cookbook.kirby.design/#/home/showcase/action-sheet) looks like:
`items: [
	{ id: '1', text: 'Option 1' },
	{ id: '2', text: 'Option 2' },
	{ id: '3', text: 'Option 3' },
]`
Some solution could be:
`items: [
	{ id: '1', text: 'Option 1', icon: 'svg-path1...' },
	{ id: '2', text: 'Option 2', icon: 'svg-path2...' },
	{ id: '3', text: 'Option 3', disabled: true },
]`
## Have you considered any alternatives?

We (Team Chi) considered the only alternative would be creating another diy-alternative-action-sheet which vould be a bad solution.

--
Nice to have:
According the styling we considered css-styling the id of the item, and maybe this would actually be a better solution?
According disabling items we considered just not displaying the item at all (not adding it to the items list). Our ux'er made the attached mock-up with items disabled. But maybe some other team need this in the future.

## Are there any additional context?

This attached mock-up is created by our ux'er and shows a rendered version of our request with added icons to the action sheet.

![image](https://user-images.githubusercontent.com/79920665/173821267-da7c8c27-81d4-4bee-b2c8-0fd484fd9935.png)

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by henrikvoetmand** [comment]
Hi Henrik, thank you for the issue :)
From UI/UX point of view, we need to work more on "Icons in action sheet"
We need to align the use of icons accross different components.
I know that theese icons is available in our design tool - but this option will be deprecated until we find the right solution.
I understand your need some UI to change category - try to look at "Spending overview" there a drawer - in combination with a list of items - is used. Please reach out to me to discuss a solution :)

**Comment by RasmusKjeldgaard** [comment]
Action Sheet is deprecated and will be removed in a later version of Kirby so we no longer accept feature requests for this component.

Instead, the newly introduced menu-component can be used to solve this use-case.
